tests/Functional/ExceptionTest.php

```diff
@@ -306,6 +306,7 @@ public function testConnectionExceptionSqLite(): void
                 $conn->executeStatement($sql);
             }
         } finally {
+            $conn->close();
```

Comment:
can be probably replaced with `gc_collect_cycles()` in `self::cleanupReadOnlyFile()` method

Reply:
I tested it now and it cannot, as the `$conn` variable is still active in the callee context.